src/silx/gui/plot/items/core.py

```diff
@@ -935,6 +938,39 @@ def setColor(self, color, copy=True):
         self._updated(ItemChangedType.COLOR)
 
 
+class LineGapColorMixIn(ItemMixInBase):
+    """Mix-in class for dashed line gap color"""
+
+    _DEFAULT_LINE_GAP_COLOR = None
+    """Default dashed line gap color of the item"""
+
+    def __init__(self):
+        self.__lineGapColor = self._DEFAULT_LINE_GAP_COLOR
+
+    def getLineGapColor(self):
```

Comment:
I hesitated to reuse shapes method naming: `get|setLineBgColor` but it's implemented slightly differently (it's not a background color here) and I choose to stick to matplotlib naming.